server.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly do not disable the linters as they help standardize your code base.